packages/systemd/systemd.spec

```diff
@@ -212,6 +212,9 @@ install -p -m 0644 %{S:3} %{buildroot}%{_cross_libdir}/systemd/journald.conf.d/j
 # with container networking by attempting to manage veth devices.
 rm -f %{buildroot}%{_cross_libdir}/systemd/network/*
 
+# Remove default target created by systemd, since we override it in the release spec
+rm -f %{buildroot}%{_cross_libdir}/systemd/{system,user}/default.target
+
```

Comment:
I'd also like to remove the stock `multi-user.target` from here, and add a copy to the release package that makes its dependencies explicit.

Then all of our targets will be packaged in the same place, and it will be somewhat easier to reason about the combined behavior.